<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEntry</class>
 <widget class="QWidget" name="DataEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="English">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>English：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEnglish">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textAttribute">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wordAttr">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>词性：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textChinese">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Chinese">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>中文：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Entry">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>录入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="End">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
